mlflow/tracking/metic_value_conversion_utils.py

```diff
@@ -0,0 +1,163 @@
+import sys
+
+from mlflow.exceptions import MlflowException
+
+
+def is_module_imported(module_name: str) -> bool:
+    return module_name in sys.modules
+
+
+def __try_get_item(x):
+    try:
+        xval = x.item()
+    except ValueError:
+        raise MlflowException(
+            f"Expected metric value to contain a single element, got shape {x.shape} instead"
```

Comment:
Can we add the INVALID_PARAMETER_VALUE `error_code` as well? Example: https://github.com/mlflow/mlflow/blob/1539d9e467bcd865c335e800dad597517b0a9bcd/mlflow/pyfunc/__init__.py#L791-L793